notebooks/215-image-inpainting/215-image-inpainting.ipynb

```diff
@@ -34,8 +37,8 @@
    "id": "c6815624-1f04-4110-8f3e-1da9f1a02cf8",
    "metadata": {},
    "source": [
-    "### Download the Model\n",
-    "Models can be downloaded with `omz_downloader` - a command-line tool for downloading models from the Open Model Zoo. `gmcnn-places2-tf` is the omz name for the considered model. You can find the names of all available models [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/index.md) and [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/index.md). The selected model comes from the public directory, which means it must be converted into Intermediate Representation (IR). This step is skipped if the model is already downloaded.\n"
+    "### Download and Unzip the Model\n",
+    "`gmcnn-places2-tf` is the omz name for the considered model. You can find the names of all available models [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/index.md) and [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/index.md). The selected model comes from the public directory, which means it must be converted into Intermediate Representation (IR). This step will be skipped if the model is already downloaded.\n"
```

Comment:
not sure that it is good idea to tell users how to download models from our s3 and provide information where other models can be found. Can we do not highlight model source in description? (just imagine that it is public model, but we store it on our server for download efficiency)

Reply:
done